### DISCUSSION TITLE
Sending an output to another host

### DISCUSSION THREAD

**Discussion by Radio-biscuit** [savonet/liquidsoap#3057] [Q&A]
Hello everyone,

It's sunny here today, so I hope you're all feeling as good as I am? Anyway ...

I need to send an output from one Liquidsoap script on one computer (happens to be Windows) to another computer (running Linux) to be an input on another Liquidsoap script. I was trying to use SRT, but it crashes for me on Windows (this is being investigated). So is there another way that I can do this without having to compress or resample the output? I'm not great with stream formats and compression knowledge, but I do know that I don't want to go from lossless to lossy just to get this across a local network.

So, should I use the Harbor server and feed that?
Or is there another protocol for Liquidsoap's output and input that I could use to send the stream (RTMP??)
Is anyone already doing this? If you are, how do you do it?

Thanks for any suggestions.

A.,

**Comment by toots** [comment]
Hi,

You should be able to use any protocol really, the key would be to use the `%audio.copy` and `%video.copy` streams of the `%ffmpeg` encoder. Internally, everything would be passing encoded data end-to-end.

See for instance: https://www.liquidsoap.info/doc-2.1.4/cookbook.html#relaying-without-re-encoding

**Comment by Radio-biscuit** [comment]
Thanks Romain!

Maybe I was trying too hard to find a way. I'll check out that part of the book again.
